arviz/plots/kdeplot.py

```diff
@@ -182,6 +182,7 @@ def plot_kde(
         rug_kwargs.setdefault("space", 0.2)
 
         plot_kwargs.setdefault("linewidth", linewidth)
+        plot_kwargs.setdefault("linestyle", "-")
```

Comment:
This is not necessary, the solid line is the default in matplotlib, thus if no linestyle is passed, the plot will have a solid line.

Reply:
plot_energy throws an exception if this line is not included.

Reply:
I thinl that the recommendation in the other comment will solve this

Reply:
Now this line can be safely removed. It is not needed afterwards anymore.